scripts/spack/configs/docker/ubuntu24/spack.yaml

```diff
@@ -9,53 +9,34 @@ spack:
     build_stage::
     - $spack/../build_stage
 
-  compilers::
-  - compiler:
-      environment: {}
-      extra_rpaths: []
-      # Axom example sidre_shocktube_F_ex doesn't link otherwise
-      flags:
-        cflags: -fPIC -pthread
-        cxxflags: -fPIC -pthread
-      modules: []
-      operating_system: ubuntu22.04
-      paths:
-        cc: /usr/bin/clang
-        cxx: /usr/bin/clang++
-        f77: /usr/bin/gfortran-11
-        fc: /usr/bin/gfortran-11
-      spec: clang@14.0.0
-      target: x86_64
-  - compiler:
-      environment: {}
-      extra_rpaths: []
-      flags:
-        cflags: -fPIC -pthread
-        cxxflags: -fPIC -pthread
-      modules: []
-      operating_system: ubuntu22.04
-      paths:
-        cc: /usr/bin/gcc
-        cxx: /usr/bin/g++
-        f77: /usr/bin/gfortran-13
-        fc: /usr/bin/gfortran-13
-      spec: gcc@13.1.0
-      target: x86_64
-  - compiler:
-      environment: {}
-      extra_rpaths: []
-      flags:
-        cflags: -fPIC -pthread
-        cxxflags: -fPIC -pthread
-      modules: []
-      operating_system: ubuntu22.04
-      paths:
-        cc: /usr/bin/gcc
-        cxx: /usr/bin/g++
-        f77: /usr/bin/gfortran-12
-        fc: /usr/bin/gfortran-12
-      spec: gcc@12.3.0
-      target: x86_64
+  toolchains:
+    clang_19:
+    - spec: '%c=llvm'
+      when: '%c'
+    - spec: '%cxx=llvm'
+      when: '%cxx'
+    - spec: '%fortran=gcc'
+      when: '%fortran'
+    - spec: '%mpich@4.0.clang_19'
+      when: '%mpi'
+    gcc_14:
```

Comment:
Do we need gcc@13 and 14?

Reply:
just checked, and the clang 19 image uses gcc 13